src/Components/Publishing/Fixtures/Articles.ts

```diff
@@ -669,15 +669,29 @@ export const MissingVerticalStandardArticle = extend({}, StandardArticle, {
   vertical: null,
 })
 
+export const VideoArticle = extend({}, StandardArticle, {
+  layout: "video",
+  media: {
+    url: '',
+    duration: '03:12',
+  },
+})
+
 export const SeriesArticle = extend({}, StandardArticle, {
   layout: "series",
   title: "The Future of Art",
+  about: "<p>Integer posuere erat a ante venenatis dapibus posuere velit aliquet. Curabitur blandit tempus porttitor. Donec ullamcorper nulla non metus auctor fringilla. Donec ullamcorper nulla non metus auctor fringilla. Vivamus sagittis lacus vel augue laoreet rutrum faucibus dolor auctor. Maecenas faucibus mollis interdum. Lorem ipsum dolor sit amet, consectetur adipiscing elit.</p><p>Duis mollis, est non commodo luctus, nisi erat porttitor ligula, eget lacinia odio sem nec elit. Curabitur blandit tempus porttitor. Sed posuere consectetur est at lobortis. Praesent commodo cursus magna, vel scelerisque nisl consectetur et. Nullam quis risus eget urna mollis ornare vel eu leo.</p><p>Donec sed odio dui. Vivamus sagittis lacus vel augue laoreet rutrum faucibus dolor auctor. Nullam quis risus eget urna mollis ornare vel eu leo. Nulla vitae elit libero, a pharetra augue.</p>",
+  related_articles: [
+    VideoArticle,
+    FeatureArticle
+  ]
```

Comment:
Nice, I like `related_articles` as the relation. It will work well with other article layouts.

I'm not too sure with the naming for `about` though. Maybe it should be `about_the_film`. It makes me wonder -- if we're planning on having something like `media` to represent the "content" of the video article, how can we do the same for a series? Especially if the "content" of a series can expand in the future. (I'm still not convinced that series should be a layout, but I need to sit on that one)